backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/flightleg/internal/BpkAccessoryTitleDetails.kt

```diff
@@ -0,0 +1,70 @@
+package net.skyscanner.backpack.compose.flightleg.internal
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.TextStyle
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@Composable
+fun BpkAccessoryTitleDetails(
```

Comment:
I'm not sure about this abstraction - IMO it makes it harder to understand how the component is structured. title, description & accessory view don't really make it clear how they're arranged and I find it easier to look at the whole component and define rows/columns within, rather than abstracting this away. 
Also this is a public function, meaning it'd be exposed as a component, but it is not a defined component. 
I know this is an abstraction you're using in sonic, but it's not really used within backpack or wider.

Reply:
yeah, I wasn't too sure about this either. I'll remove it and move everything to the FlightLeg component

Reply:
can you remove this please?

Reply:
🤦.
Thanks! forgot to remove it